common/src/main/java/com/wynntils/services/lootrunpaths/LootrunRenderer.java

```diff
@@ -138,6 +140,10 @@ private static void renderChests(PoseStack poseStack, LootrunPathInstance lootru
         float blue = ((float) FastColor.ARGB32.blue(color)) / 255;
 
         for (BlockPos chest : chests) {
+            // Wynncraft requested that chest highlights are not rendered on these blocks
+            BlockState block = McUtils.mc().level.getBlockState(chest);
+            if (block.is(Blocks.BARRIER) || block.is(Blocks.AIR)) continue;
```

Comment:
Does it mean that a chest that is not present at the moment will not be rendered? I'm not sure that is always a good thing. I have loaded lootruns that I downloaded just to get inspiration on where to look for chests.

... but then again, it was some time ago I played Wynncraft, maybe that does not make sense anymore with the new personal chest cooldown.

Reply:
It won't render the outline for barriers or air, it was requested to stop things like this where barriers are being shown
![image](https://github.com/Wynntils/Artemis/assets/8337467/e7f47b47-c539-44ce-8347-0522517d4c23)

Reply:
Since you can place "chests" to be highlighted anywhere, even if they aren't actually chests